docs/source/index.md

```diff
@@ -0,0 +1,204 @@
+# poliastro - Astrodynamics in Python
+
+```{eval-rst}
```

Comment:
Doesn't MyST support images?

Reply:
MyST does support image but as seen from this [example](https://myst-parser.readthedocs.io/en/latest/using/syntax.html#how-directives-parse-content), it doesn't support arguments such as `width`, `alignment`, etc. So to overcome this problem, in all the changes `{eval-rst}` directives have been kept to ensure that the image retains its intended nature :)

Reply:
I have migrated it ... Don't worry now it's native `MyST` syntax ... Thank You very much for changing my mind though this will take time as I will need to skim through each document to change it :)